### DISCUSSION TITLE
Trying to Grock incremental server rendering

### DISCUSSION THREAD

**Discussion by pkellner** [vercel/next.js#15962] [Help] [answered]
I get the idea that as a new page shows up, it gets built into static JavaScript and then somehow deployed as static, but I seem to have missed the implementation detail.  What is the tech that generates this page? Is that something that has to be run against netlify or vercel? Is that something that I can setup in a docker container? Sorry if everyone totally gets this and I'm the only lost puppy out here.  Feel free to point me at some doc. I've read the nextjs docs multiple times and it's just not clear to me what is happening on the server to make this magic happen.

(BTW, I'm really not a fan of the prayer icon when posting for help, feels a little condescending)

**Comment by pkellner** [comment]
Very lonely here

**Comment by mcsdevv** [comment, marked as answer]
Hey @pkellner, as I mentioned yesterday on Discord, we're going to be documenting this in full soon, sorry for the delay!

For now, this is a very good resource for understanding and explaining what's going on: https://web.dev/stale-while-revalidate/
